store/rootmulti/store.go

```diff
@@ -907,27 +907,36 @@ func (rs *Store) buildCommitInfo(version int64) *types.CommitInfo {
 }
 
 // RollbackToVersion delete the versions after `target` and update the latest version.
-func (rs *Store) RollbackToVersion(target int64) int64 {
-	if target < 0 {
-		panic("Negative rollback target")
+func (rs *Store) RollbackToVersion(target int64) error {
```

Comment:
LGTM. Isn't there other metadata we need to flush too?

Reply:
> Isn't there other metadata we need to flush too?

is there?